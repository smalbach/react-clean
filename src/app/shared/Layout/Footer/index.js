import React from "react";

const Footer = () => (
  <div className="Footer">
    <p className="Footer-title" data-cy="footer-title">
      React app
    </p>
    <p className="Footer-copy">Feel free to copy, made with love</p>
  </div>
);

export default Footer;
