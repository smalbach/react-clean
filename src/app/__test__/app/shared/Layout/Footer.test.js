import React from "react";
import { mount } from "enzyme";
import Footer from "../../../../shared/Layout/Footer";
import { create } from "react-test-renderer";

describe("<Footer>", () => {
  const footer = mount(<Footer />);
  test("Render of footer", () => {
    expect(footer.length).toEqual(1);
  });

  test("Title render", () => {
    expect(footer.find(".Footer-title").text()).toEqual("React app");
  });
  test("UI  footer", () => {
    const footer = create(<Footer />);
    expect(footer.toJSON).toMatchSnapshot();
  });
});
