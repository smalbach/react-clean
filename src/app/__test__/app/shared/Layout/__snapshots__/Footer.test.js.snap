// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> Test UI Component footer 1`] = `[Function]`;
